docs/pages/faq/react-native-styling-buttons.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Styling a React Native Button'
+---
+
+React Native exports a [`<Button />`](https://reactnative.dev/docs/button) component that exposes the native button element for Android, iOS, and the web. The `<Button />` component accepts `title` and `onPress` props, however it does not accept a `style` prop, which makes it hard to customize the style. The closest we can get to styling a `<Button />` exported from React Native is with the `color` prop. Below is an example of two buttons on Android, iOS, and the web. The first button is the default `<Button />` and the second is another default `<Button />` with its `color` prop set to `"red".
+
+![default-button](https://user-images.githubusercontent.com/6455018/112760134-f82c9c80-8fc3-11eb-86fd-59a4d9abf68b.png)
+
+To create a button with a custom style, we can to turn to the [`<Pressable />`](https://reactnative.dev/docs/pressable) component.`<Pressable />`let's us fully customize the appearance of a pressable element (like a button), in addition to allowing us to customize it's behavior. Here's an example of using`<Pressable />` to create a button component:
+
+```jsx
+import React from 'react';
+import { Text, View, StyleSheet, Pressable } from 'react-native';
+export default function Button(props) {
+  const { onPress, title = 'Save' } = props;
+  return (
+    <Pressable style={styles.button} onPress={onPress}>
+      <Text style={styles.text}>{title}</Text>
+    </Pressable>
+  );
+}
```

Comment:
```suggestion
const Button = ({ onPress, title = 'Save' }) => (
  <Pressable style={styles.button} onPress={onPress}>
    <Text style={styles.text}>{title}</Text>
  </Pressable>
);
export default Button;
```
What do you think about this format?

Reply:
That format works as well. I personally find the keyword `function` nice to read. I also usually name the argument `props` and destructure it below, so that if this were to be typed with TypeScript, we'd get this syntax (which feels readable to me):

```tsx
type Props = {
  prop1: string;
  prop2: number;
  prop3: boolean;
}

function Button (props: Props) {
  const { prop1, prop2, prop3 } = props;
  // ...
}
``` 

The pattern I personally find slightly harder to read quickly is this:

```tsx
const Button =  ({ prop1, prop2, prop3 }: { prop1: string, prop2: number, prop3: boolean }) => {
  // ...
}
```

That said, either is fine. Is there a pattern in docs? If arrow functions with destructured args are the norm, we should change this to be consistent!

Reply:
Agree with @jonsamp about using the function keyword. It's clearer than `const X = () => ...`. Also, with object literals, `{ fn() {} }` is often clearer than `{ fn: () => {} }`. We intentionally don't configure our linter to require arrow functions wherever possible. Arrow functions make the most sense in callbacks where it's more important to close over the environment.